### DISCUSSION TITLE
Takım görevleri 4 - Şifreli mesajı çözüp cevaplayın

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/BilsemPython#91] [Göster ve Söyle]
pejak peueiji öföelaijeö. aljacıfılıv üveöe zu ziö ridöeji kepaicıö. zu kepaiı bövcüısel polöa saııkjaöılıva gib ıasıı pafjakaüal öföelaijeöil ipikjeöili üile zu reıijce ridöejeüeöeı üavkalıvı ipsiüoöuk. aülı vakalca galei saııkca ojcufuluvu ca zejiösil. zu ridöeüi ejje ki bövcülüv üoıpa oöoeöak üavaöaı kı bövcülüv zu ca zelik ibil ölekji ejzesse.

**Comment by akgembi12** [comment]
Burada her harfin karşılığı aynı olamaz çünkü `ıasıı` kelimesinin Türkçe'de herhangi bi karşılığı yok (aynı iki harfle bitip ilk harfinin de bu harflerle aynı olduğu bir kelime yok)

**Comment by tolga0102** [comment]
keöhaza öföesmelik, zel oisol üıjaljaöı saııkılıl ıaosalıüım. saııkıkca ıasıı pafjakaül ıirijeöcel gib ziö daajiües eöpseökeüel ede(edeovaeüjal) acjı ıiricel boı riıaüesbiüim.

**Reply by sonsuzus** [reply]
aljacık, sereııüö eceöik sojea

**Comment by akgembi12** [comment]
gmaak ziv ml sge waü saiıkı mjaöai ridöeüi bnvcüi. saiıkcal ziö aöiacarıl zijpekcel aüöıjka cuöuku uaö, mlulja ijesirike eebeaefiv. ridöelil kalsıfılı ejje bnvcük aka pmlöa sük ridöeüi oömeöakja zujcui
